src/linear_algebra/finite_dimensional.lean

```diff
@@ -209,34 +315,36 @@ lemma ker_eq_bot_iff_range_eq_top [finite_dimensional K V] {f : V →ₗ[K] V} :
   f.ker = ⊥ ↔ f.range = ⊤ :=
 by rw [range_eq_top, ker_eq_bot, injective_iff_surjective]
 
+/-- In a finite-dimensional space, if linear maps are inverse to each other on one side then they
+are also inverse to each other on the other side. -/
 lemma mul_eq_one_of_mul_eq_one [finite_dimensional K V] {f g : V →ₗ[K] V} (hfg : f * g = 1) :
```

Comment:
Why are these lemmas stated in terms of multiplication instead of composition?

Reply:
I haven't touched these lemmas, so you should ask the original author. Since `V →ₗ[K] V` is an algebra, multiplication or composition are the same, but I guess the notation is much nicer with multiplication...

Reply:
Yeah... but I can imagine that users might hope that `comp_eq_id_of...` autocompletes to something useful...

Reply:
I have added a version formulated with `comp` for the iff version, but not for this version which is only an auxiliary statement for the proof of the iff.